flink-queryable-state/flink-queryable-state-client-java/src/main/java/org/apache/flink/queryablestate/client/QueryableStateClient.java

```diff
@@ -268,7 +268,7 @@ public ExecutionConfig setExecutionConfig(ExecutionConfig config) {
 			final String queryableStateName,
 			final int keyHashCode,
 			final byte[] serializedKeyAndNamespace) {
-		LOG.info("Sending State Request to {}.", remoteAddress);
+		LOG.debug("Sending State Request to {}.", remoteAddress);
```

Comment:
We should go for:
```
if (LOG.isDebugEnabled) {
    ....
}
```

Reply:
alright, will fix and merge it.

Reply:
actually, it is fine this way. 
`LOG.debug()` is overloaded with non-varargs implementations.

Reply:
I see.